docker-compose.malware-scanner.yml

```diff
@@ -0,0 +1,51 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+
+  clamav:
+    image: clamav/clamav:stable_base
+    container_name: clamav
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/clamav/clamav/defs:/var/lib/clamav
+      - ./docker/clamav/clamd.conf:/etc/clamav/clamd.conf:ro
+    expose:
+      - 3310 # NEVER expose this outside of the local network!
+    deploy:
+      resources:
+        limits:
+          cpus: '${CLAMAV_CPU:-0.50}'
+    networks:
+      shared:
+        ipv4_address: 10.10.10.100
```

Comment:
I think we should change the ip address here. Blocker is at `.102`. That should change to `100` and this should be `120` and up? (I have `110` in another PR). The idea is to give it like 10 IP address to future proof  it against extensions to the service right? We could also just make `blocker` 120 but then I have to update my PR

Reply:
I moved blocker to 110 and kept clamav at 100 and malware-scanner at 101.